src/scaffoldmaker/utils/shieldmesh.py

```diff
@@ -129,8 +130,8 @@ def getTriplePoints(self, n3):
             tx, td1 = sampleCubicHermiteCurves(
                 [self.px[n3][n2a][n1c], self.px[n3][n2c][n1b]],[[(-self.pd1[n3][n2a][n1c][c] - self.pd3[n3][n2a][n1c][c]) for c in range(3)], self.pd1[n3][n2c][n1b]],2, arcLengthDerivatives=True)[0:2]
             ltx.append(tx[1])
-            tx, td1 = sampleCubicHermiteCurves(
-                [ self.px[n3][n2a][n1b], self.px[n3][n2c][n1c] ], [ [-self.pd3[n3][n2a][n1b][c] for c in range(3)], [ (self.pd1[n3][n2c][n1c][c] + self.pd3[n3][n2c][n1c][c]) for c in range(3) ] ], 2, arcLengthDerivatives = True)[0:2]
+            # tx, td1 = sampleCubicHermiteCurves(
+            #     [ self.px[n3][n2a][n1b], self.px[n3][n2c][n1c] ], [ [-self.pd3[n3][n2a][n1b][c] for c in range(3)], [ (self.pd1[n3][n2c][n1c][c] + self.pd3[n3][n2c][n1c][c]) for c in range(3) ] ], 2, arcLengthDerivatives = True)[0:2]
```

Comment:
Why are these commented out here & in subsequent lines?

Reply:
I do not have points between the rim and triple points at this stage, and they will be created later on. Also, I am not using it for finding the triple point. I can remove the commented lines?